templates/workloads/services/worker/manifest.yml

```diff
@@ -29,13 +29,12 @@ cpu: {{.CPU}}       # Number of CPU units for the task.
 memory: {{.Memory}}    # Amount of memory in MiB used by the task.
 count: {{.Count.Value}}       # Number of tasks that should be running in your service.
 exec: true     # Enable running commands in your container.
-
 {{- if .Subscribe}}{{- if .Subscribe.Topics}}
```

Comment:
What do you think of creating a commented block if there is no subscribe setting present? for example:
```yaml
{{- if .Subscribe }} 
{{- if .Subscribe.Topics}}
# The events will be available in an SQS queue via the env var $COPILOT_SQS_QUEUE_ARN
... what you already have
{{- else }}
# You can register to topics from other services.
# The events will be available in an SQS queue via the env var $COPILOT_SQS_QUEUE_ARN
# subscribe:
#   topics: 
#   - name: topic-from-another-service
#      service: another-service
```
And also adding a comment saying which env var they can use to retrieve the events from the queue 🙏

Reply:
That's a good idea so that how to use subscribe is clear! Adjusted.